modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/structures/FieldGroups.testcase

```diff
@@ -236,12 +234,59 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98077. As a Developer, I want to the ability to nest fields"
-	@ignore = "true"
 	@priority = "4"
 	test SetFieldGroupAsCollapsibleAndExecuteOnAWebContent {
-		property portal.acceptance = "false";
+		var structureName = "WC Structure Name";
```

Comment:
Hey @brunoquei2105 , the same here... I don't think you need to remove this property.